music21/articulations.py

```diff
@@ -501,7 +501,12 @@ class OpenString(Bowing):
     pass
 
 class StringIndication(Bowing):
-    pass
+    def __init__(self):
+        super().__init__()
+        self.number = ''
```

Comment:
.number should be None or an int, and be able to be passed into the constructor as with fingering above.  Since the class is being developed, documentation and a doctest are required.  (see Fingering above).